srcpkgs/stgit/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'stgit'
+pkgname=stgit
+version=1.5
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools git"
```

Comment:
Doesn't need git build time, but needs git and python3 runtime. Please add `depends`.